docs/user/rstudio/ide/guide/accessibility/screen-reader.qmd

```diff
@@ -2,15 +2,9 @@
 title: "Screen Reader"
 ---
 
-Adapted from: <https://support.rstudio.com/hc/en-us/articles/360045612413-RStudio-Screen-Reader-Support>
-
 This article aims to provide screen-reader users with some effective strategies for RStudio IDE navigation.
 
-Some features discussed in this document require RStudio 1.3 or newer. In a few cases you need to know if you are using RStudio Desktop or RStudio Workbench / RStudio Server. If RStudio is loaded in your web browser, then you are using RStudio Workbench or RStudio Server; otherwise you have RStudio Desktop.
-
-## Screen Reader
-
-Please see the [article on RStudio Screen Reader support](https://support.rstudio.com/hc/en-us/articles/360045612413-RStudio-Screen-Reader-Support) for details on using the RStudio IDE with a screen reader.
+Some features discussed in this document require RStudio 1.3 or newer. In a few cases you need to know if you are using RStudio Desktop or Posit Workbench / RStudio Server. If RStudio is loaded in your web browser, then you are using Posit Workbench or RStudio Server; otherwise you have RStudio Desktop.
```

Comment:
Nit
```suggestion
Some features discussed in this document require RStudio 1.3, or newer. In a few cases, you need to know if you are using RStudio Desktop or Posit Workbench / RStudio Server. If RStudio is loaded in your web browser, then you are using Posit Workbench or RStudio Server; otherwise, you have RStudio Desktop.
```

Reply:
Changed